UPGRADE-7.3.md

```diff
@@ -189,3 +189,8 @@ VarDumper
 
  * Deprecate `ResourceCaster::castCurl()`, `ResourceCaster::castGd()` and `ResourceCaster::castOpensslX509()`
  * Mark all casters as `@internal`
+
+HttpClient
+----------
+
+* Deprecate passing an instance of `StoreInterface` as `$cache` argument to `CachingHttpClient` constructor
```

Comment:
```suggestion
 * Deprecate passing an instance of `StoreInterface` as `$cache` argument to `CachingHttpClient` constructor
```

Also please add the `HttpClient` entry in the alphabetical order with other components 🙂

Reply:
adressed in https://github.com/symfony/symfony/pull/59576/commits/23141cbc8bbc7196d4ca4cac6d18599984459d21